README.md

```diff
@@ -277,6 +280,56 @@ You can use the following query:
 }
 ```
 
+## Gatsby cloud and preview environment setup
+
+### Setup
+
+To enable content sync in [Gatsby cloud](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/deploying-to-gatsby-cloud/) you need to create two webhooks in your Strapi project:
+
+- [Build webhook](https://support.gatsbyjs.com/hc/en-us/articles/360052324394-Build-and-Preview-Webhooks)
+
+![webhook setup](./assets/webhook.png)
+
+- [Preview webhook](https://support.gatsbyjs.com/hc/en-us/articles/360052324394-Build-and-Preview-Webhooks)
+
+At this point each time you create an entry the webhooks will trigger a new build a deploy your new Gatsby site.
+
+- In the Site settings, Environment variables fill the:
+  - Build variables with the following:
+    - STRAPI_API_URL with the url of your deployed Strapi application
+    - STRAPI_TOKEN with your build API TOKEN
+  - Preview variables:
+    - STRAPI_API_URL with the url of your deployed Strapi application
+    - STRAPI_TOKEN with your preview API TOKEN
+
+### Enabling Content Sync
+
+#### Installing the @strapi/plugin-gatsby-preview
+
+In order to enable Gatsby Content Sync and integrate it in your Strapi CMS you need to install the `@strapi/plugin-gatsby-preview` in your Strapi project:
+
+##### Using yarn
+
+```sh
+cd my-strapi-app
+
+yarn add @strapi/plugin-gatsby-preview
+```
+
+##### Using npm
+
+```sh
+cd my-strapi-app
+npm install --save @strapi/plugin-gatsby-preview
+```
+
+#### Configurations
+
+Once the plugin is installed you will need to configure it in the plugin's settings section.
+
+- In the Collection types or the Single Types tab enable when enabling the **preview** it will inject a button in the content manager edit view of the corresponding collection type. So, after creating an entry (draft or published) when clicking on the **Open Gatsby preview** button you will be redirected to the Gatsby preview page
```

Comment:
```suggestion
- In the Collection types or the Single Types tab, when enabling the **preview**, it will inject a button in the content manager edit view of the corresponding content type. So, after creating an entry (draft or published), clicking on the **Open Gatsby preview** button will redirect you to the Gatsby preview page.
```